### ISSUE TITLE
[BUG] <Adding anchor tag in the logo>

### ISSUE THREAD

**Issue by RamakrushnaBiswal** [krishnaacharyaa/wanderlust#362]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Anchor tag not adding in the logo

### Expected Behavior

here add a anchor tag and when a user click upon it redirect to home page

### Reproduction steps

```bash
1. Go to '...'
2. Click on 'while clicking'
3. Scroll down to '....'
4. See error
```

### Screenshots

https://github.com/krishnaacharyaa/wanderlust/assets/125277258/7e6c703d-a93d-440e-9ede-dc27765694a0

### Browsers

Chrome, Microsoft Edge

### OS

Windows

**Comment by RamakrushnaBiswal** [comment]
hi @krishnaacharyaa can you assign it to me under GSSCO 24,i want to work upon it.

**Comment by krishnaacharyaa** [comment]
okay @RamakrushnaBiswal

**Comment by RamakrushnaBiswal** [comment]
hi @krishnaacharyaa i face some issue while commit to my branch
![image](https://github.com/krishnaacharyaa/wanderlust/assets/125277258/d53e2a4a-127c-44cb-85b9-24707468180c)

**Comment by RamakrushnaBiswal** [comment]
@krishnaacharyaa kindly review the PR #375

**Comment by RamakrushnaBiswal** [comment]
issue solved PR #375
